src/bitriseAPI/triggerMultipleBuilds.js

```diff
@@ -0,0 +1,23 @@
+const {consoleRed, consoleGreen} = require('../utils/consoleColors');
+const {createPayload} = require("../utils/createPayload");
+const {triggerBuild} = require("./triggerBuild");
+
+
+/**
+ * @param builds A list with each builds' desired branch, workflow and environmental variables.
+ */
+const triggerBuildMultipleBuilds = async (builds) => {
```

Comment:
Could you change that to `triggerMultipleBuilds?`

Reply:
You are absolutely right, my mistake.